include/deal.II/base/memory_consumption.h

```diff
@@ -631,6 +641,16 @@ namespace MemoryConsumption
   template <typename T>
   inline
   std::size_t
+  memory_consumption (const std_cxx11::unique_ptr<T> &)
+  {
+    return sizeof(std_cxx11::unique_ptr<T>);
```

Comment:
It's a fair question. I simply opted for consistency with the other pointer types.

The old code already added the memory of the pointer plus the memory of the object pointed to. I didn't change that in this patch (nor should it be changed).